reviews mobile

chester669 opened the pull request:
finished reviews section for a mobile

chester669 committed:
reviews mobile

finished reviews section for a mobile

DorianGray0 reviewed (changes requested):
я тобі зкинув усі зміни в slack , передивись їх буль ласка та виправ

chester669 committed:
web-table-phone

fixed a previous mistakes ( only pictures still at public cause i don't see img folder ).
